python/mxnet/ndarray.py

```diff
@@ -1066,12 +1086,12 @@ def binary_ndarray_function(lhs, rhs, out=None):
                                        c_array(NDArrayHandle, (lhs.handle, rhs.handle)),
                                        c_array(mx_float, ()),
                                        c_array(NDArrayHandle, (out.handle,)),
-                                       ctypes.c_int(0),
-                                       c_array(ctypes.c_char_p, []),
-                                       c_array(ctypes.c_char_p, [])))
+                                       ctypes.c_int(len(kwargs)), \
+                                       c_array(ctypes.c_char_p, kwargs.keys()), \
+                                       c_array(ctypes.c_char_p, [str(i) for i in kwargs.values()])))
```

Comment:
I found I need to use `key.encode('ascii')` and `str(i).encode('ascii')` to make the travis test pass. Related to python3? https://github.com/dmlc/mxnet/pull/2292/